### PR TITLE
{bio}[foss/2019a] StringTie v1.3.5

### PR DESCRIPTION
**Pull request by arielzn**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@arielzn Does `StringTie` need MPI or BLAS/LAPACK?

If not, we should pull it down to a subtoolchain like `GCC`?

File: easybuild/easyconfigs/s/StringTie/StringTie-1.3.5-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+source_urls = ['http://ccb.jhu.edu/software/%(namelower)s/dl/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f16ec07d4869b7656bc1a9b6eb662fd9404c122d18431c845e93d810840c5db9']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+]
+
+# release option must be given to 'make' otherwise a DEBUG version is built by default
+buildopts = " release"
```
Comment (boegel) [inline easybuild/easyconfigs/s/StringTie/StringTie-1.3.5-foss-2019a.eb]: @arielzn We don't have this (or the `zlib` dep) in `StringTie-1.3.5-foss-2018b.eb`, is it worth fixing that there?

Comment (arielzn) [reply easybuild/easyconfigs/s/StringTie/StringTie-1.3.5-foss-2019a.eb]: the `buildopts = release` i would say yes, by running just `make` the code is built with debug symbols and forces `-O0` 

You can verify in some build logs with the upstream version, that it's dumped something like `WARNING: built DEBUG version [much slower], use "make clean release" for a faster, optimized version of the program.`

The missing zlib dep, gave me issues when trying to build in a CentOS 6 with a really old system zlib. Actually this made me look at the logs and notice the other message at some point.

Comment (arielzn) [reply easybuild/easyconfigs/s/StringTie/StringTie-1.3.5-foss-2019a.eb]: it's also specified in the [README](https://github.com/gpertea/stringtie/blob/master/README.md)

**Comment by arielzn** [comment]
>  Does `StringTie` need MPI or BLAS/LAPACK?

it looks that it doesn't and a build with GCCcore-8.2.0 produces something working.

can I just change the .eb file in this PR by one using GCCcore toolchain?
or you close and i submit another ?

**Comment by akesandgren** [comment]
Changing this one is a bit awkward, not hard but includes adding a new .eb and removing the existing one and changing the title.

So I'd say open a new one (for GCCcore) with a ref to this and then close this one.

File: easybuild/easyconfigs/s/StringTie/StringTie-1.3.5-foss-2019a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/StringTie/StringTie-1.3.5-foss-2019a.eb]: fixed in existing StringTie 1.3.5 easyconfig via #7845

**Pull request closed without merging**